data/src/main/java/com/anytypeio/anytype/data/auth/repo/AuthCache.kt

```diff
@@ -26,4 +25,7 @@ interface AuthCache {
 
     suspend fun getNetworkMode(): NetworkModeConfig
     suspend fun setNetworkMode(modeConfig: NetworkModeConfig)
+
+    suspend fun updateReserveMultiplexLibrary(useReserve: Boolean)
```

Comment:
could be a part of network config:

 ```
suspend fun getNetworkMode(): NetworkModeConfig
 suspend fun setNetworkMode(modeConfig: NetworkModeConfig)
```

for instance, by adding a boolean field into `NetworkModeConfig`